frontend/src/components/MainNav.tsx

```diff
@@ -1,13 +1,19 @@
 import { useAuth0 } from '@auth0/auth0-react';
 import { Button } from './ui/button';
 import UsernameMenu from './UsernameMenu';
+import { Link } from 'react-router-dom';
 
 const MainNav = () => {
   const { loginWithRedirect, isAuthenticated } = useAuth0();
   return (
     <span className="flex space-x-2 items-center">
       {isAuthenticated ? (
-        <UsernameMenu />
+        <>
+          <Link to="/order-status" className="font-bold hover:text-orange-500">
+            Order Status
+          </Link>
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding aria-label for accessibility</b></div>
<div id="fix">

Consider adding an `aria-label` to the `Link` component for better accessibility. This could help screen readers provide more context about the link's purpose.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -12,3 +12,3 @@
 -          <Link to="/order-status" className="font-bold hover:text-orange-500">
 +          <Link to="/order-status" className="font-bold hover:text-orange-500" aria-label="Check Order Status">
              Order Status

```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/Food-Ordering-App/pull/49#issuecomment-2439875949>#873af7</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged